packages/polkadot/src/__snapshots__/hydration.moonbeam.test.ts.snap

```diff
@@ -0,0 +1,1237 @@
+                  "unlimited": null,
+                },
+              },
+            },
+            {
+              "depositAsset": {
+                "assets": {
+                  "wild": {
+                    "allCounted": 1,
+                  },
+                },
+                "beneficiary": {
+                  "interior": {
+                    "x1": {
+                      "accountKey20": {
+                        "key": "0xf24ff3a9cf04c71dbc94d0b566f7a27b94566cac",
+                        "network": null,
+                      },
+                    },
+                  },
+                  "parents": 0,
+                },
+              },
+            },
+          ],
+        },
+      },
+    ],
+  },
+]
+`;
+
+exports[`hydration & moonbeam > hydration transfer DOT to moonbeam > route chain xcm events 1`] = `
+[
+  {
+    "data": {
+      "id": "(hash)",
+      "origin": {
+        "Ump": {
+          "Para": "(rounded 2000)",
+        },
+      },
+      "success": true,
+      "weightUsed": {
+        "proofSize": "(rounded 7200)",
+        "refTime": "(rounded 540000000)",
+      },
+    },
+    "method": "Processed",
+    "section": "messageQueue",
+  },
+]
+`;
+
+exports[`hydration & moonbeam > hydration transfer DOT to moonbeam > to chain xcm events 1`] = `
+[
+  {
+    "data": {
+      "id": "(hash)",
+      "origin": "Parent",
+      "success": true,
+      "weightUsed": {
+        "proofSize": "(rounded 26000)",
+        "refTime": "(rounded 690000000)",
+      },
+    },
+    "method": "Processed",
+    "section": "messageQueue",
+  },
+]
+`;
+
+exports[`hydration & moonbeam > hydration transfer DOT to moonbeam > tx events 1`] = `
+[
+  {
+    "data": {
+      "assets": [
+        {
+          "fun": {
+            "Fungible": 1000000000000,
+          },
+          "id": {
+            "interior": "Here",
+            "parents": 1,
+          },
+        },
+      ],
+      "dest": {
+        "interior": {
+          "X2": [
+            {
+              "Parachain": "(rounded 2000)",
+            },
+            {
+              "AccountKey20": {
+                "key": "0xf24ff3a9cf04c71dbc94d0b566f7a27b94566cac",
+                "network": null,
+              },
+            },
+          ],
+        },
+        "parents": 1,
+      },
+      "fee": {
+        "fun": {
+          "Fungible": 1000000000000,
+        },
+        "id": {
+          "interior": "Here",
+          "parents": 1,
+        },
+      },
+      "sender": "7Lh1DEaxk8TNVU7snC3UkSv7N7NHb7BUuEkYZBwrzfrPNQE3",
+    },
+    "method": "TransferredAssets",
+    "section": "xTokens",
+  },
+]
+`;
+
+exports[`hydration & moonbeam > hydration transfer GLMR to moonbeam > balance on from chain 1`] = `
+{
+  "free": 0,
+  "frozen": 0,
+  "reserved": 0,
+}
+`;
+
+exports[`hydration & moonbeam > hydration transfer GLMR to moonbeam > balance on to chain 1`] = `
+{
+  "consumers": 0,
+  "data": {
+    "flags": 0,
+    "free": 0,
```

Comment:
is this expected?

Reply:
You are right, the transaction indeed fails, and the asset is trapped on Moonbeam.

Is there a way to enable detailed xcm logs so I could troubleshoot the issue easily? When we use chopsticks, we usually use `runtime-log-level: 5` in the runtime yaml config file, but in this repo I could not find the same, only to increase log level with `LOG_LEVEL=debug`, but that does not lead to detailed xcm runtime logs.

Reply:
use `RUNTIME_LOG_LEVEL=5`

https://github.com/open-web3-stack/polkadot-ecosystem-tests/blob/master/README.md#environment-variables